create a docs page for the default parameters of pvcompare (#28)
greco-project/pvcompare: docs/parameters.rst

=========================================================
Parameters of PVCompare: Definitions and Default Values
=========================================================

Within the pvcompare/pvcompare/data/ directory, two separate cateogies of inputs can be observed.

1. MVS parameters (found in the CSVs within the data/mvs_inputs/csv_elements/ directory)
2. PVCompare-specific parameters (found in the CSVs within the data/inputs directory)

------------------
1. MVS Parameters
------------------

> As PVCompare makes use of the `MVS <https://github.com/rl-institut/mvs_eland>`_ tool, the definitions of all the

Reviewer: ```suggestion
As PVCompare makes use of the `Multi-vector Simulation Tool "MVS" <https://github.com/rl-institut/mvs_eland>`_ tool, the definitions of all the
```

> relevant parameters of MVS can be found `here <https://mvs-eland.readthedocs.io/en/latest/MVS_parameters.html>`_. Note

Reviewer: ```suggestion
relevant parameters of MVS can be found `here <https://mvs-eland.readthedocs.io/en/latest/MVS_parameters.html>`_. 
```

> relevant parameters of MVS can be found `here <https://mvs-eland.readthedocs.io/en/latest/MVS_parameters.html>`_. Note

Reviewer: ```suggestion
relevant parameters of MVS can be found in its `documentation <https://mvs-eland.readthedocs.io/en/latest/MVS_parameters.html>`_. Note
```

> that some parameters of MVS (for instance, all parameters concerning a diesel generator) are not applicable for PVCompare.

Reviewer: ```suggestion

```
I would remove this sentence as I think that with pvcompare you can use all components of MVS - we just don't do it.

    10. **dispatch_price**: currency/kWh, 0 (for all components)
    11. **optimizeCap**: bool, True (for all components)
    12. **outflow_direction**: str
        a. **storage_charge_controller_in**: ESS Li-Ion
        b. **storage_charge_controller_out**: Electricity
        c. **solar_inverter_01**: Electricity (if there are more solar inverters, this value applies for them as well)
    13. **energyVector**: str, Electricity (same for all the components)
    14. **type_oemof**: str, transformer (same for all the components)
    15. **unit**: str, kW (applies to all the components)
* energyProduction.csv:
    1. **age_installed**: year, 0 (for all the components)
    2. **development_costs**: currency, 0 (**TO BE DECIDED**)
    3. **specific_costs**: currency/unit, (**TO BE DECIDED**)
    4. **file_name**: {optional} str
        a. **pv_plant_01**: si_180_31.csv
        b. **pv_plant_02**: cpv_180_31.csv
        c. **pv_plant_03**: cpv_90_90.csv
    5. **installedCap**: kWp, 0.0 (for all components)

>     6. **maximumCap**: {optional} kWp

Reviewer: The maximum capacity is calculated by pvcompare - should we leave this out or keep is as default as it is the default that is calculated in case building parameters are not changed?

Reviewer 2: I'm not sure what you mean. The parameter should still be listed in this document. But the value is marked as "optional" which i changed now to "auto_calc". An explanation is given on the top that the value is calculated automatically.

        a. **pv_plant_01**: 25454.87
        b. **pv_plant_02**: 55835.702
        c. **pv_plant_03**: 23929.586
    7. **label**: {optional} str
        a. **pv_plant_01**: PV si_180_31
        b. **pv_plant_02**: PV cpv_180_31
        c. **pv_plant_03**: PV cpv_90_90
    8. **lifetime**: year, 25 (for all the components)

>     9. **specific_costs_om**: currency/unit, 50 (same for all the components; 50 €/kWp is the value that is arrived at after accounting for the yearly inspection and cleaning. Here is the detailed `explaination <https://github.com/greco-project/pvcompare/issues/13>`_.)

Reviewer: ```suggestion
    9. **specific_costs_om**: currency/unit, 50 (same for all the components; 50 €/kWp is the value that is arrived at after accounting for the yearly inspection and cleaning. Here is the detailed `explanation <https://github.com/greco-project/pvcompare/issues/13>`_.)
```

    10. **dispatch_price**: currency/kWh

        a. **storage capacity**: NA (does not apply)

        b. **input power** and **output power**: 0
    11. **soc_initial**: None or factor

        a. **storage capacity**: None

        b. **input power** and **output power**: NA
    12. **soc_max**: factor

        a. **storage capacity**: 0.8 (Took the Fronius 4.5 battery which has a rated capacity 4.5 kW, but 3.6 kW is the usable capacity.So SoC max would be 80% or 0.8.)

        b. **input power** and **output power**: NA
    13. **soc_min**: factor

        a. **storage capacity**: 0.1 (Figure from this research `article <https://www.sciencedirect.com/science/article/pii/S0378775319310043>`_.)

        b. **input power** and **output power**: NA
    14. **unit**: str
        a. **storage capacity**: kWh

        b. **input power** and **output power**: kW

---------------------------------
2. PVCompare-specific parameters
---------------------------------

In order to run PVCompare, a number of input parameters are needed; many of which are stored in csv files with default values in "pvcompare/inputs/".

> Further a few parameters must be filled in when calling the main function. These parameters will not be of interest here. For more information see the functions description.  The following list will give a brief introduction into the description of the csv files and the source of the given default parameters.

Reviewer: ```suggestion
Further a few parameters must be filled in when calling the main function. These parameters will not be of interest here. For more information see the function's description.  The following list will give a brief introduction into the description of the csv files and the source of the given default parameters.
```
Do you refer to main() ? Could you add a link please?
